lsp-mode.el

```diff
@@ -6490,28 +6490,41 @@ should return the command to start the LS server."
   "Create LSP stdio connection named name.
 LOCAL-COMMAND is either list of strings, string or function which
 returns the command to execute."
+  ;; 2.5.0-pre (as built from native-comp branch before M Albinus released tramp-2.5)
+  ;; worked fine
+  (defvar tramp-version)
+  (defvar tramp-connection-properties)
+  (when (version< "2.5.0-pre" tramp-version)
+    (lsp-warn
+     "Your tramp version - %s - might fail to work with remote LSP. Update to tramp-2.5 for tested reliability improvements"
```

Comment:
can you change the wording to mention that you can install that version from elpa?

Reply:
BTW, when I tested this function with tramp 2.5.0 from elpa, it showed the warning.

Reply:
Thanks for testing my branch! I think I have the arguments in the wrong order